src/api/consumer/lib/test-inline.js

```diff
@@ -6,7 +6,9 @@ import Bit from '../../../consumer/component';
 
 export async function test(id: string, verbose: boolean = true): Promise<Bit> {
```

Comment:
do we call this file? 
it looks like an old file (inline folder is dead).
If we do call it, we should rename it.